woocommerce/woocommerce
# CYS > Update copy to remove the duplicate "try to include"

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR remove the duplicated "Try to include" text.




Closes https://github.com/woocommerce/woocommerce/issues/42948



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have WooCommerce Beta Tester installed.
2. Enable the `customize-store` feature flag in Tools -> WCA Test Helper -> Features.
3. Go to WooCommerce -> Home and click on `Customize your store` task.
4. Make sure you only see the "Try to include" text once:

| Before | After |
|--------|--------|
| <img width="550" alt="Screenshot 2023-12-19 at 16 57 34" src="https://github.com/woocommerce/woocommerce/assets/186112/c86c2bfb-2d29-4d8f-83c3-519e5c093bd5"> |<img width="550" alt="Screenshot 2023-12-19 at 16 57 04" src="https://github.com/woocommerce/woocommerce/assets/186112/ff091dd7-3776-4efe-b94a-52ee39082c1b">| 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Remove the duplicated "Try to include" copy on the first CYS screen.
#### Comment 

</details>
